wavepool/views.py

```diff
@@ -13,9 +13,20 @@ def front_page(request):
             archive: the rest of the newsposts, sorted by most recent
     """
     template = loader.get_template('wavepool/frontpage.html')
-    cover_story = NewsPost.objects.all().order_by('?').first()
-    top_stories = NewsPost.objects.all().order_by('?')[:3]
-    other_stories = NewsPost.objects.all().order_by('?')
+    cover_story = None
+    top_stories = []
+    other_stories = []
+    newsposts = NewsPost.objects.all().order_by('publish_date')
```

Comment:
Can we filter the query instead of getting all?
Maybe a model helper function at some point